index.bs

```diff
@@ -547,17 +547,85 @@ navigator.getVRDisplays().then(function (displays) {
 </pre>
 </div>
 
+## VRDisplayEventReason ## {#interface-vrdisplayeventreason}
+
+<pre class="idl">
+interface VRDisplayEventReason {
+  readonly attribute unsigned short code;
+  const unsigned short VR_DISPLAY_NAVIGATION = 1;
+  const unsigned short VR_DISPLAY_MOUNTED = 2;
+  const unsigned short VR_DISPLAY_UNMOUNTED = 3;
+};
+</pre>
+
+### Attributes ### {#vrdisplayeventreason-attributes}
+
+<dfn attribute for="VRDisplayEventReason">code</dfn>
+The code describing the reason for this event, described below.
+
+### Reason Codes ### {#vrdisplayeventreason-codes}
```

Comment:
I'm good with that! It's also consistent with how we handle the left and right eyes, now that I think about it.

I had modeled this after [MediaError ](https://dev.w3.org/html5/spec-author-view/video.html#error-codes) because it was the first thing that came to mind when looking for a pre-existing API that handled a similar situation. I have no particular love for that API, though.